samples/apps/copilot-chat-app/webapp/src/libs/services/DocumentImportService.ts

```diff
@@ -8,19 +8,21 @@ export class DocumentImportService extends BaseService {
         userId: string,
         userName: string,
         chatId: string,
-        document: File,
+        documents: FileList,
         accessToken: string,
     ) => {
         const formData = new FormData();
         formData.append('userId', userId);
         formData.append('userName', userName);
         formData.append('chatId', chatId);
         formData.append('documentScope', 'Chat');
-        formData.append('formFile', document);
+        for (let i = 0; i < documents.length; i++) {
```

Comment:
nit: prefer `for...of` where possible (suggested by linter)
```suggestion
        for (const document of documents) {
```